Foody/Home/Category.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Category.swift
+//  Foody
+//
+//  Created by Ezgi  on 30.01.2023.
+//
+
+import Foundation
+import UIKit
+
+struct Category{
```

Comment:
Struct kullanman cok hosuma gitti. Ama  Category structindan instance yaratirken unlem isareti istemediginiz icin  asagidaki gibi tanimlarsan unlem isaretine gerek kalmaz. 
```suggestion
struct Category{
   let title: String?
    let image: UIImage?
}
```